<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plural Sample</string>
  </property>
  <widget class="QLabel" name="label0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label101">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label111">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">dummy</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
